commons/src/main/java/org/infinispan/commons/dataconversion/CompatModeEncoder.java

```diff
@@ -33,7 +33,7 @@ public Object toStorage(Object content) {
    public Object fromStorage(Object content) {
```

Comment:
The toStorage/fromStorage methods should be annotated with ```@Override```. This is missing in some other implementations of ```Encoder```.